src/ssh/azext_ssh/_params.py

```diff
@@ -37,10 +40,16 @@ def load_arguments(self, _):
                    help='Folder where new generated keys will be stored.')
         c.argument('cert_file', options_list=['--certificate-file', '-c'], help='Path to certificate file')
         c.argument('port', options_list=['--port'], help='SSH port')
+        c.argument('ssh_client_folder', options_list=['--ssh-client-folder'],
```

Comment:
We may need to update the documentation page (if any). please check with sandeep.

Reply:
I'll confirm, but it seems that the documentation for "az ssh" gets updated automatically once the changes get merged. https://docs.microsoft.com/en-us/cli/azure/ssh?view=azure-cli-latest